spring-batch-docs/asciidoc/repeat.adoc

```diff
@@ -167,9 +166,10 @@ Users might need to implement their own completion policies for more
 If there is an exception thrown inside a
     `RepeatCallback`, the
     `RepeatTemplate` consults an
-    `ExceptionHandler` which can decide whether or not to
+    `ExceptionHandler`, which can decide whether or not to
     re-throw the exception.
 
+The following listing shows the `ExceptionHandler` interface definition:
 
 [source, java]
```

Comment:
Noticed that handleException throws a Throwable not a RuntimeException.   
I can change this on Merge

Reply:
Yes, please. Thanks again.